src/github/logic.py

```diff
@@ -217,20 +218,42 @@ def _is_pull_request_ready_for_automerge(pull_request: PullRequest) -> bool:
         )
 
     if pull_request_has_label(pull_request, AutomergeLabel.AFTER_TESTS.value):
```

Comment:
I think we would run into a possible issue of this comment getting added over and over again in a loop and probably need a way to prevent that

Reply:
We could do a check to see if the comment already exists!

Reply:
sounds good! Looks like you have done this recently and should be pretty straightforward :)